cron/auth02mozdef.py

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/env python
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2016 Mozilla Corporation
+# Author: gdestuynder@mozilla.com
+
+# Imports auth0.com logs into MozDef
+
+import hjson
+import sys
+import requests
+import mozdef_client as mozdef
+try:
+    import urllib.parse
+    quote_url = urllib.parse.quote
+except ImportError:
+    #Well hello there python2 user!
+    import urllib
+    quote_url = urllib.quote
+
+class DotDict(dict):
```

Comment:
Agreed, I think eventually they could have certain functions/classes exposed to the cron jobs, or like you said, some sort of "shell" that you overwrite a "run" or "execute" function to perform the custom code.